rewrite-java-21/src/main/java/org/openrewrite/java/isolated/ReloadableJava21TypeMapping.java

```diff
@@ -634,20 +622,8 @@ public JavaType.Primitive primitive(TypeTag tag) {
                 if (!methodType.thrown.isEmpty()) {
                     exceptionTypes = new ArrayList<>(methodType.thrown.size());
                     for (Type exceptionType : methodType.thrown) {
-                        JavaType.FullyQualified javaType = TypeUtils.asFullyQualified(type(exceptionType));
```

Comment:
This logic appears to be a holdover from a time when we had `JavaType.Cyclic`, so I believe is safe to simplify now.